tls/_common/prf.py

```diff
@@ -8,6 +8,63 @@
 from cryptography.hazmat.primitives.hmac import HMAC
 
 
+"""
+This file implements the TLS Psuedo-Random Function as specified by Section 5
+of RFC 5246 (https://tools.ietf.org/html/rfc5246#section-5). The section
+states:
+
+
+   The TLS record layer uses a keyed Message Authentication Code (MAC) to
```

Comment:
In addition to moving this up farther, this will render better if it's marked as a preformatted block.  I think you just have to add `..` to the line before (if this is all like, 4 spaces).